backend/app/Entities/Visit.php

```diff
@@ -4,25 +4,9 @@
 
 namespace App\Entities;
 
-use Carbon\Carbon;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\BelongsTo;
 
-/**
- * Class Visit
- * @package App\Entities
- * @property int $id
- * @property Carbon $visit_time
- * @property string $ip_address
- * @property Session $session
- * @property int $session_id
- * @property Page $page
- * @property int $page_id
- * @property Visitor $visitor
- * @property int $visitor_id
- * @property GeoPosition $geo_position
```

Comment:
it was okay, such notation is good to understand what properties in the model are used